_config/config.yml

```diff
@@ -7,3 +7,9 @@ SilverStripe\Admin\LeftAndMain:
     - 'silverstripe/tagfield:client/dist/js/bundle.js'
   extra_requirements_css:
     - 'silverstripe/tagfield:client/dist/styles/bundle.css'
+
+---
+Name: tagfieldconfig
+---
+SilverStripe\TagField\StringTagField:
+  immediate_write_enabled: true
```

Comment:
Instead of setting this in a yml file, please set it in the class itself and add a [deprecation notice](https://docs.silverstripe.org/en/4/contributing/release_process/#deprecation)

Reply:
Ok, PR updated. I guess it's ok to deprecate it for the next major version 3.0.0?